test/mysql.autoupdate.test.js

```diff
@@ -176,4 +211,25 @@ describe('MySQL connector', function() {
       done();
     });
   });
+
+  it('"mysql.columnName" is updated with correct name on create table', function (done) {
```

Comment:
@rlloyd2001 Thanks for the contribution, I'll try to help you out on the way. The linter runs automatically when you run `npm test` or you can run it manually via `npm run lint` -- see https://github.com/strongloop/loopback-connector-mysql/blob/master/package.json#L8.